man/networkctl.xml

```diff
@@ -355,6 +355,31 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         which match the file are reconfigured.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>
+          <command>edit</command>
+          <replaceable>FILE|DEVICE…</replaceable>
+        </term>
+        <listitem><para>Edit network configuration files. If no network config file matching
+        the given name is found, a new one will be created under <filename>/etc/</filename>.
+        Specially, if the name is prefixed by <literal>@</literal>, it will be treated as
+        a network interface, and editing will be performed on the associated network file, if any.
+
+        If <option>--drop-in=</option> is specified, edit the drop-in file instead of
```

Comment:
You need to close the paragraph with `<para>` and then open a new one.

You can preview the man page with `build/man/man networkctl`.